platform/security/core/security-core-services/src/main/java/ddf/security/audit/impl/SecurityLoggerImpl.java

```diff
@@ -53,14 +56,16 @@ public final class SecurityLoggerImpl implements ddf.security.audit.SecurityLogg
 
   private static final String EXTRA_ATTRIBUTES_PROP = "security.logger.extra_attributes";
 
-  public static final String TRACE_ID = "trace-id";
-
   private static final String LOOPBACK_ADDRESS = "127.0.0.1";
 
   private final SubjectOperations subjectOperations;
 
-  public SecurityLoggerImpl(SubjectOperations subjectOperations) {
+  private final List<AuditPropertiesPlugin> auditPropertiesPlugins;
+
+  public SecurityLoggerImpl(
+      SubjectOperations subjectOperations, List<AuditPropertiesPlugin> auditPropertiesPlugins) {
```

Comment:
✏️ Since the AuditPropertiesPlugins are not required, I would add them as a getter/setter rather than modifying the constructor.  That way, you don't have to change several of the other places in the code that specifically construct this class like the tests and `SecurityPolicyConfigurator`.